src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -737,6 +770,186 @@ end probability_measure -- namespace
+
+lemma normalize_eq_inv_mass_smul_of_nonzero [inhabited α] (μ : finite_measure α) (nonzero : μ ≠ 0) :
+  μ.normalize.to_finite_measure = (μ.mass)⁻¹ • μ :=
+begin
+  nth_rewrite 2 μ.self_eq_mass_smul_normalize,
+  rw ← smul_assoc,
+  simp only [μ.mass_nonzero_iff.mpr nonzero, algebra.id.smul_eq_mul,
+             inv_mul_cancel, ne.def, not_false_iff, one_smul],
+end
+
+lemma coe_normalize_eq_of_nonzero [inhabited α] (nonzero : μ ≠ 0) :
+  (μ.normalize : measure α) = (μ.mass)⁻¹ • μ :=
+begin
+  ext1 s s_mble,
+  simp only [← μ.normalize.ennreal_coe_fn_eq_coe_fn_to_measure s,
+             μ.normalize_eq_of_nonzero nonzero s, ennreal.coe_mul,
+             ennreal_coe_fn_eq_coe_fn_to_measure, measure.coe_nnreal_smul_apply],
+end
+
+@[simp] lemma _root_.probability_measure.to_finite_measure_normalize_eq_self
+  [inhabited α] (μ : probability_measure α) :
+  μ.to_finite_measure.normalize = μ :=
+begin
+  ext s s_mble,
+  rw μ.to_finite_measure.normalize_eq_of_nonzero μ.to_finite_measure_nonzero s,
+  simp only [probability_measure.mass_to_finite_measure, inv_one, one_mul],
+  refl,
+end
+
+variables [topological_space α]
+
+lemma test_against_nn_eq_mass_mul [inhabited α] (f : α →ᵇ ℝ≥0) :
+  μ.test_against_nn f = μ.mass * μ.normalize.to_finite_measure.test_against_nn f :=
+begin
+  nth_rewrite 0 μ.self_eq_mass_smul_normalize,
+  rw μ.normalize.to_finite_measure.smul_test_against_nn_apply μ.mass f,
+  refl,
+end
+
+lemma normalize_test_against_nn
+  [inhabited α] (μ : finite_measure α) (nonzero : μ ≠ 0) (f : α →ᵇ ℝ≥0) :
+  μ.normalize.to_finite_measure.test_against_nn f = (μ.mass)⁻¹ * μ.test_against_nn f :=
+by simp [μ.test_against_nn_eq_mass_mul, μ.mass_nonzero_iff.mpr nonzero]
+
+variables [opens_measurable_space α]
+
+variables {μ}
+
+lemma tendsto_mass_of_tendsto
+  {γ : Type*} {F : filter γ} {μs : γ → finite_measure α} (h : tendsto μs F (𝓝 μ)) :
+  tendsto (λ i, (μs i).mass) F (𝓝 μ.mass) :=
+(continuous_mass.tendsto μ).comp h
+
+lemma tendsto_zero_test_against_nn_of_tendsto_zero_mass
+  {γ : Type*} {F : filter γ} {μs : γ → finite_measure α}
+  (mass_lim : tendsto (λ i, (μs i).mass) F (𝓝 0)) (f : α →ᵇ ℝ≥0) :
+  tendsto (λ i, (μs i).test_against_nn f) F (𝓝 0) :=
+begin
+  apply tendsto_iff_dist_tendsto_zero.mpr,
+  have obs := λ i, (μs i).test_against_nn_lipschitz_estimate f 0,
+  simp_rw [test_against_nn_zero, zero_add] at obs,
+  simp_rw (show ∀ i, dist ((μs i).test_against_nn f) 0 = (μs i).test_against_nn f,
+    by simp only [dist_nndist, nnreal.nndist_zero_eq_val', eq_self_iff_true,
+                  implies_true_iff]),
+  refine squeeze_zero (λ i, nnreal.coe_nonneg _) obs _,
+  simp_rw nnreal.coe_mul,
+  have lim_pair : tendsto (λ i, (⟨nndist f 0, (μs i).mass⟩ : ℝ × ℝ)) F (𝓝 (⟨nndist f 0, 0⟩)),
+  { refine (prod.tendsto_iff _ _).mpr ⟨tendsto_const_nhds, _⟩,
+    exact (nnreal.continuous_coe.tendsto 0).comp mass_lim, },
+  have key := tendsto_mul.comp lim_pair,
+  rwa mul_zero at key,
+end
```

Comment:
these two lemmas are not really about normalized measures. Could you move them to a more relevant place?